mergedSchema.js

```diff
@@ -0,0 +1,46 @@
+import { mergeSchemas, introspectSchema, makeRemoteExecutableSchema } from "graphql-tools"
+import { createHttpLink } from "apollo-link-http"
+import fetch from "node-fetch"
+
+import localSchema from "./schema"
+
+export default async function mergedSchema() {
+  const convectionLink = createHttpLink({
+    fetch,
+    uri: process.env.CONVECTION_GRAPH_URL,
+    headers: {
+      Authorization: `Bearer ${process.env.CONVECTION_TOKEN}`,
+    },
+  })
+
+  const convectionSchema = await makeRemoteExecutableSchema({
+    schema: await introspectSchema(convectionLink),
+    link: convectionLink,
+  })
+
+  const linkTypeDefs = `
+    extend type Submission {
+      artist: Artist
+    }
+  `
+
+  return mergeSchemas({
+    schemas: [localSchema, convectionSchema, linkTypeDefs],
+    // Prefer others over the local MP schema.
+    onTypeConflict: (_leftType, rightType) => {
+      // console.warn(`[!] Type collision ${rightType}`)
```

Comment:
@orta There’s currently a bunch of consignments related type name conflicts, presumably because you’ve manually added these to MP’s schema before:

```
19:04:06 web.1       |  [!] Type collision ID
19:04:06 web.1       |  [!] Type collision Submission
19:04:06 web.1       |  [!] Type collision String
19:04:06 web.1       |  [!] Type collision Asset
19:04:06 web.1       |  [!] Type collision JSON
19:04:06 web.1       |  [!] Type collision Boolean
19:04:06 web.1       |  [!] Type collision Int
```

The `ID`, `String`, `Boolean`, and `Int` ones are built-in though, it seems weird that `graphql-tools` would trigger this callback for those 🤔